doc/rqg_guide.md

```diff
@@ -0,0 +1,2823 @@
+# 1. Test Objective
+This guide is introduced to how to perform a RQG test. The RQG test is amed to test CUBRID stability especially in terms of server crash. It generates various of querys(update/delete/insert/select) automatically based on grammar files and executes these queries in parallel.
+# 2. Test Tools

```

Comment:
Merge `#2` and `#3` chapters.
And adjust order as first installation, then usage.